data_standards.md

```diff
@@ -51,27 +51,27 @@ To be advised by ACS Data Governance Framework.
 
 #### Directory structure:  
 
-/g/data/ia39/australian-climate-service/\<**status**\>/\<**activity**\>/\<**product**\>/\<**domain**\>/\<**RCM-institution**\>/\<**GCM-model-name**\>/\<**CMIP6-experiment-name**\>/\<**CMIP6-ensemble-member**\>/\<**RCM-model-name**\>/\<**RCM-version-ID**\>/\<**frequency-or-category**\>/\<**variable-name**>
+/g/data/ia39/australian-climate-service/\<**status**\>/\<**activity**\>/\<**product**\>/\<**domain**\>/\<**RCM-institution**\>/\<**GCM-model-name**\>/\<**experiment-name**\>/\<**ensemble-member**\>/\<**RCM-model-name**\>/\<**RCM-version-ID**\>/\<**frequency-or-category**\>/\<**variable-name**>
 
 #### Filenaming:  
 
-\<**variable-name**\>\_\<**domain**\>\_\<**GCM-model-name**\>\_\<**CMIP6-experiment-name**\>\_\<**CMIP6-ensemble-member**\>\_\<**RCM-model-name**\>\_\<**RCM-version-id**\>\_\<**frequency-or-category**\>[\_\<**StartTime-EndTime**\>].nc
+\<**variable-name**\>\_\<**domain**\>\_\<**GCM-model-name**\>\_\<**experiment-name**\>\_\<**ensemble-member**\>\_\<**RCM-model-name**\>\_\<**RCM-version-id**\>\_\<**frequency-or-category**\>[\_\<**StartTime-EndTime**\>].nc
   
 #### Controlled Vocabulary:
 
 | **key** = | [*restriction*:] { item1 item2 ... }  | notes |
 | ------------ | ------------ | --------- |
-| **status** | { `test-data`, `release` }  | |
-| **activity** | { `CORDEX-CMIP6` }  | |
+| **status** | { `test-data`, `staged`, `pre-release`, `release` }  | |
```

Comment:
I'm not sure about the need for both 'staged' and 'pre-release'; how are these different?

Reply:
Meeting outcome: add `staged` but don't need `pre-release`